documentation/en/chapter_14.md

```diff
@@ -0,0 +1,971 @@
+    <td>-200</td>
+    <td>0</td>
+    <td><code>2001::200:9:0:0</code></td>
+ </tr>
+  <tr align="left">
+    <td>Node-J</td>
+   <td>
+        <ul>
+            <li><code>2001::200:a:0:0/128 dev #7</code></li>
+            <li><code>2001::200:1:0:0/128 via fe80::200:9:0:0 dev #7</code></li>
+        </ul> 
+    </td>
+    <td>100</td>
+    <td>-200</td>
+    <td>10</td>
+    <td><code>2001::200:a:0:0</code></td>
+ </tr>
+  <tr align="left">
+    <td>Node-K</td>
+   <td>
+        <ul>
+            <li><code>2001::200:b:0:0/128 dev #7</code></li>
+            <li><code>2001::200:1:0:0/128 via fe80::200:a:0:0 dev #7</code></li>
+        </ul> 
+    </td>
+    <td>200</td>
+    <td>-200</td>
+    <td>0</td>
+    <td><code>2001::200:b:0:0</code></td>
+ </tr>
+  <tr align="left">
+    <td>Node-L</td>
+   <td>
+        <ul>
+            <li><code>2001::200:c:0:0/128 dev #7</code></li>
+            <li><code>2001::200:1:0:0/128 via fe80::200:8:0:0 dev #7</code></li>
+            <li><code>2001::200:10:0:0/128 via fe80::200:10:0:0 dev #7</code></li>
+        </ul> 
+    </td>
+    <td>300</td>
+    <td>-200</td>
+    <td>0</td>
+    <td><code>2001::200:c:0:0</code></td>
+ </tr>
+  <tr align="left">
+    <td>Node-M</td>
+   <td>
+        <ul>
+            <li>2001::200:d:0:0/128 dev #7</li>
+            <li>2001::200:1:0:0/128 via fe80::200:9:0:0 dev #7</li>
+        </ul> 
+    </td>
+    <td>0</td>
+    <td>-300</td>
+    <td>0</td>
+    <td><code>2001::200:d:0:0</code></td>
+ </tr>
+  <tr align="left">
+    <td>Node-N</td>
+   <td>
+        <ul>
+            <li><code>2001::200:e:0:0/128 dev #7</code></li>
+            <li><code>2001::200:1:0:0/128 via fe80::200:d:0:0 dev #7</code></li>
+        </ul> 
+    </td>
+    <td>100</td>
+    <td>-300</td>
+    <td>0</td>
+    <td><code>2001::200:e:0:0</code></td>
+ </tr>
+  <tr align="left">
+    <td>Node-O</td>
+   <td>
+        <ul>
+            <li><code>2001::200:f:0:0/128 dev #7</code></li>
+            <li><code>2001::200:1:0:0/128 via fe80::200:e:0:0 dev #7</code></li>
+        </ul> 
+    </td>
+    <td>200</td>
+    <td>-300</td>
+    <td>0</td>
+    <td><code>2001::200:f:0:0</code></td>
+ </tr>
+  <tr align="left">
+    <td>Node-P</td>
+   <td>
+        <ul>
+            <li><code>2001::200:10:0:0/128 dev #7</code></li>
+            <li><code>2001::200:1:0:0/128 via fe80::200:c:0:0 dev #7</code></li>
+        </ul> 
+    </td>
+    <td>300</td>
+    <td>-300</td>
+    <td>0</td>
+    <td><code>2001::200:10:0:0</code></td>
+ </tr>
+</table>
+</div>
+<br>
+
+
+
+## Experimental Results
+
+We evaluated the efficiency of the AODV protocol via simulations in renode, As mentioned earlier, the overall number of control packets generated by __AODVV2__ is more higher than for standard __AODV__, due to in this latest version there aren't intermediate __RREP__ from nodes that know the path to the destination, this approach in order to avoid __loop_free__ generations and ensure reliable paths. 
```

Comment:
```suggestion
We evaluated the efficiency of the AODVv2 protocol via simulations in renode, As mentioned earlier, the overall number of control packets generated by __AODVV2__ is more higher than for standard __AODV__, due to in this latest version there aren't intermediate __RREP__ from nodes that know the path to the destination, this approach in order to avoid __loop_free__ generations and ensure reliable paths. 
```